libs/styles/src/lib/scss/components/_status-badge.scss

```diff
@@ -14,7 +14,7 @@
   line-height: get-line-height(100);
   white-space: nowrap;
 
-  .badge-icon {
+  #{$base}.badge-icon {
```

Comment:
This change breaks the appearance of the status badge icon – the class name stops being applied to the markup. See [a visual  comparison here](https://www.chromatic.com/test?appId=6373995e3f280e239470296d&id=64905006abaee7d4b89a607f).

Even though there's a linter warning here, it perhaps makes sense to leave the selector as it was in this PR since 1) the way this warning is fixed breaks styles for `cvi-status-badge` Sass component and 2) the change is unrelated to textarea component which is the scope of the PR.